pennylane_lightning/lightning_qubit.py

```diff
@@ -423,12 +449,7 @@ def compute_vjp(self, dy, jac, num=None):
         else:
             raise TypeError(f"Unsupported complex Type: {dtype}")
 
-        vjp_tensor = VJP.compute_vjp_from_jac(
-            math.reshape(jac, [-1]),
-            dy_row,
-            num,
-            num_params,
-        )
+        vjp_tensor = VJP.compute_vjp_from_jac(math.reshape(jac, [-1]), dy_row, num, num_params)
```

Comment:
Probably the version problem. I updated black and it makes every function call single line... May add `format:off` and `on`.

Reply:
Ah! This could be the case. Not sure using `format:off` and `on` frequently is a good practice. Perhaps we can follow black suggestions for now and figure something out later.

Reply:
It was the version problem. But I found I was using an old version of black (instead of newer) as I was using Python 3.7 in my venv (somehow...). I changed it to Python 3.8 and black does not complain about this.